variables/Four Column Regex.tpl

```diff
@@ -0,0 +1,342 @@
+        ],
+        "displayName": "",
+        "simpleValueType": true,
+        "name": "convertFalseTo",
+        "type": "TEXT"
+      }
+    ]
+  }
+]
+
+
+___SANDBOXED_JS_FOR_WEB_TEMPLATE___
+
+const makeString = require('makeString');
+const inputVar = makeString(data.inputVar);
+const table = data.regexTable;
+const defaultValue = data.defaultValue;
+//Advanced Settings Values
+const ignoreCase = data.ignoreCase;
+const fullMatch = data.fullMatch;
+const capRep = data.captureReplace;
+//Format Value Values
+const convertCaseTo = data.convertCaseTo;
+const convertCase = data.convertCase;
+const convertNullTo = data.convertNullTo;
+const convertNull = data.convertNull;
+const convertUndefinedTo = data.convertUndefinedTo;
+const convertUndefined = data.convertUndefined;
+const convertTrueTo = data.convertTrueTo;
+const convertTrue = data.convertTrue;
+const convertFalseTo = data.convertFalseTo;
+const convertFalse = data.convertFalse;
+//Declare Variables
+  //If you need to change your default delimter, do it here.
+var val = defaultValue !== undefined ? defaultValue.split('|') : defaultValue,
+    a,
+    b;
+if(typeof(table) !=='undefined'){
+  //Itterate through the table
+  for(a=0;a<table.length;a++){
+    if(ignoreCase===true){
+      //Convert regex and input to lowercase
+      var input = inputVar.toLowerCase(),
+          re = table[a].inputVal.toLowerCase();
+    } else{
+      var input = inputVar,
+          re = table[a].inputVal;
+    }
+    if(input.match(re) !== null){
+      if (fullMatch===true){
+        if(input.match(re)[0]==input){
+          val=[table[a].output1,table[a].output2,table[a].output3];
+          if (capRep === true){
+            val = [table[a].output1,table[a].output2,table[a].output3];
+            for(b=0;b<val.length;b++){
+              if(val[b].match('\\$\\d+') !== null){
+                val[b] = input.match(re)[val[b].match('\\d+')[0]];
+              }
+            }
+          }
+          break;
+        } else {
+          continue;
+        }
+      } else {
+        val = [table[a].output1,table[a].output2,table[a].output3];
+        if (capRep === true){
+          for(a=b;b<val.length;b++){
+            if(val[b].match('\\$\\d+') !== null){
+              val[b] = input.match(re)[val[b].match('\\d+')[0]];
+            }
+          }
+        }
+        break;
+      }
+    }
+  }
+}
+
+if (val !== undefined){
+  //Convert variables
+  for(a=0;a<val.length;a++){
+    if(convertUndefined === true && val[a]=='undefined'){
+      val[a]=convertUndefinedTo;
+      continue;
+    }
+    if(convertNull === true && val[a]=='null'){
+      val[a]=convertNullTo;
+      continue;
+    }
+    if(convertTrue === true && val[a]=='true'){
+      val[a]=convertTrueTo;
+      continue;
+    }
+    if(convertFalse === true && val[a]=='false'){
+      val[a]=convertFalseTo;
+      continue;
+    }
+    if(convertTrue === true && val[a]=='true'){
+      val[a]=convertTrueTo;
+      continue;
+    }
+  }
```

Comment:
Same comment here as with the other template - you're now checking if the fields have the *string* values of "undefined", "null", "true" etc. so you're not actually converting `undefined` to anything, you're converting `"undefined"` to something. It's not really the same thing, and it can throw users off who expect value formatting to only apply to actual `undefined` rather than its string counterpart.